app/views/homepages/_category.html.erb

```diff
@@ -0,0 +1,15 @@
+<section class="top-ten__list-container" >
+  <h2 class="top-ten__header">Top <%= category.capitalize %>s</h2>
+  <% works = Work.top_ten("#{category}")  %>
```

Comment:
You should avoid calling model methods directly in views.

Instead you should set an instance variable in the appropriate controller.